spec/factories.rb

```diff
@@ -15,8 +15,8 @@
     role "sysadmin"
   end
 
-  factory :threshold_user, :parent => :admin_user do
-    role "threshold"
+  factory :moderator_user, :parent => :admin_user do
```

Comment:
This moderator factory is probably redundant. Should I declare admin factory with

```factory :admin, aliases: [:moderator] do```

and get rid of this?